packages/vue/src/composables/useClerk.ts

```diff
@@ -0,0 +1,26 @@
+import { useClerkContext } from './useClerkContext';
+
+/**
+ * Returns the [`Clerk`](https://clerk.com/docs/references/javascript/clerk/clerk) instance which provides
+ * low-level access to Clerk's functionality, enabling custom authentication UI and flows.
+ *
+ * @example
+ * A simple example:
```

Comment:
We can remove the "A simple example" from the `@example` blocks (also in other files of this PR) because the tag will already mark it as such + the word "simple" should probably be avoided as for some folks it might indeed not be simple (and thus discouraging)

Reply:
Updated! thank you